docs/docs/04-standard-library/helpers/api-reference.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
what is this file? can it be removed?

Reply:
I think so since it's automatically generated.